timemachine/fe/rbfe.py

```diff
@@ -164,11 +176,21 @@ def setup_initial_states(st, host_config, temperature, lambda_schedule, seed):
 
     initial_states = []
 
+    # check that the lambda schedule is monotonically increasing.
+    assert np.all(np.diff(lambda_schedule) > 0)
```

Comment:
Should we use this? `from timemachine.fe.lambda_schedule import validate_lambda_schedule`

Reply:
1) API's not quite the same, i.e. `num_windows` is not specified
2) we don't strictly require that the schedule start at 0 and 1 at the end (some of our benchmark tests just set the lambda schedule to [0.1])